src/modules/CSVModule.js

```
@@ -7,12 +8,17 @@ class CSVModule {
     this.data = parse(fs.readFileSync(csvFilePath), { columns: true });
   }
 
-  async get(key, value) {
+  async get(key, value, fromDate, toDate) {
     logger.info(`Get csvModule info by key '${key}'`);
     // return all rows if key and value aren't provided
     if (!key && !value) return this.data;
-    const result = this.data.filter((d) => d[key] === value);
+    let result = this.data.filter((d) => d[key] === value);
     if (result.length === 0) throw new ReferenceError(`CSV Record with provided key '${key}' and value was not found`);
```

DP-_NOT_-your-problem-now: We should consider whether or not we want an error thrown here, or if a logger error would be sufficient.

---

That's a good point. Now that we better handle returning an empty bundle, maybe a `logger.error` is sufficient or maybe it's something we should really throw an error for. I'll make a note to discuss at backlog grooming.